core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -648,7 +655,7 @@ protected Scope visitDelete(Delete node, Optional<Scope> scope)
         protected Scope visitAnalyze(Analyze node, Optional<Scope> scope)
         {
             QualifiedObjectName tableName = createQualifiedObjectName(session, node, node.getTableName());
-            analysis.setUpdateType("ANALYZE", tableName, Optional.empty());
+            analysis.setUpdateType("ANALYZE", tableName, Optional.empty(), ImmutableList.of());
```

Comment:
can we analyze subset of columns? I don't think we should pass empty list, I think we should not pass a list at all (Optional.empty())